packages/ruleset/test/utils/root-document.js

```diff
@@ -168,7 +168,7 @@ module.exports = {
         },
       },
     },
-    '/v1/drinks/menu': {
+    '/v1/drinkmenu': {
```

Comment:
Picky but we should probably go ahead and call it `drink_menu` since it's two words

Reply:
haha, good catch.

Reply:
fixed